packages/components/src/view/stories/index.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { View } from '..';
+
+const meta: ComponentMeta< typeof View > = {
+	component: View,
+	title: 'Components (Experimental)/View',
+	argTypes: {
+		as: { control: { type: null } },
+		children: { control: { type: 'text' } },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template: ComponentStory< typeof View > = ( args ) => {
+	return <View { ...args } />;
+};
```

Comment:
I just added a quick story so I could verify in Docs view that I didn't mess up the props.

Reply:
Do you think it's worth keeping this story ?

Reply:
No strong opinion. But `View` is actually a publicly exported component (albeit experimental), so I think it makes sense for it to have a place in Storybook, if only to see what it is.